### PR TITLE
feat: add user email by default to invoice recipients

### PR DESCRIPTION
**Pull request by jcamposmk**
### **Add user email by default at InvoiceRecipients**
**Jira Ticket:** https://makingsense.atlassian.net/browse/DAT-925

**Description**
When the user does not have added recipients. The email user is used by default.

### CODE REVIEW

**Comment by jhoffman-ms** [comment]
👍

File: src/components/Plans/Checkout/PurchaseSummary/InvoiceRecipients/index.js [hunk 1]
```diff
@@ -17,42 +17,59 @@ export const InvoiceRecipients = InjectAppServices(
...
-          setRecipients(response.value.recipients);
+        const { success, value } = await dopplerBillingUserApiClient.getInvoiceRecipientsData();
+        if (success) {
+          if (value.recipients?.length > 0) {
+            setRecipients(value.recipients);
+          } else {
+            submitEditRecipients({
+              [fieldNames.editRecipients]: [userEmail],
+            });
+          }
```
Comment (andresmoschini) [inline src/components/Plans/Checkout/PurchaseSummary/InvoiceRecipients/index.js:58]: I did not read the logic, but, in general, we are trying to reduce nesting. What do you think about this:

```suggestion
        const { success, value } = await dopplerBillingUserApiClient.getInvoiceRecipientsData();
        if (success && value.recipients?.length > 0) {
          setRecipients(value.recipients);
        } else if (success) {
          submitEditRecipients({
            [fieldNames.editRecipients]: [userEmail],
          });
```

Comment (jcamposmk) [reply src/components/Plans/Checkout/PurchaseSummary/InvoiceRecipients/index.js:58]: it's ok! better!

**Pull request merged**